src/retriever.jl

```diff
@@ -1,5 +1,50 @@
 module retriever
+using PyCall
 
+@pyimport retriever as rt
 # package code goes here
+function check_for_updates()
+    rt.check_for_updates()
+end
+
+function dataset_names()
+    rt.dataset_names()
+end
+
+function download(dataset; path::AbstractString="./", quite::Bool=false, subdir::Bool=false, use_cache::Bool=false, debug::Bool=false)
+    rt.download(dataset, path, quite, subdir, debug, use_cache)
+end
+
+function install_csv(dataset; table_name::AbstractString=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_csv(dataset, table_name, compile, debug, quite, use_cache)
+end
+
+function install_mysql(dataset; user::AbstractString="root", password::AbstractString="", host::AbstractString="localhost", port::UInt16=3306, database_name::AbstractString=nothing, table_name::AbstractString=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_mysql(dataset, user, password, host, port, database_name, table_name, compile, debug, quite, use_cache)
+end
+
+function install_postgres(dataset; user::AbstractString="postgres", password::AbstractString="", host::AbstractString="localhost", port::UInt16=5432, database::AbstractString="postgres", database_name::AbstractString=nothing, table_name::AbstractString=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_postgres(dataset, user, password, host, port, database, database_name, table_name, compile, debug, quite, use_cache)
+end
+
+function install_sqlite(dataset; file::AbstractString=nothing, table_name::AbstractString=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_sqlite(dataset, file, table_name, compile, debug, quite, use_cache)
+end
+
+function install_msaccess(dataset; file::AbstractString=nothing, table_name::AbstractString=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_msaccess(dataset, file, table_name, compile, debug, quite, use_cache)
+end
+
+function install_json(dataset; table_name::Bool=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_json(dataset, table_name, compile, debug, quite, use_cache)
+end
+
+function install_xml(dataset; table_name::Bool=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_xml(dataset, table_name, compile, debug, quite, use_cache)
+end
+
+function reset_retriever(;scope::AbstractString="all")
```

Comment:
Yes, `;` is used separate [keyword arguments](https://docs.julialang.org/en/release-0.5/manual/functions/).